app/index.php

```diff
@@ -2,11 +2,127 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
-    <link rel="stylesheet" type="text/css" href="css/style.css" >
+    <title>Bread and Ifits</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+            integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+            crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+            integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+            crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+            integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+            crossorigin="anonymous"></script>
     <script type="text/javascript" src="js/dist/main.js"></script>
 </head>
 <body>
+<nav class="nav nav-pills flex-column flex-sm-row">
+    <a class="col-2 flex-sm-fill text-sm-center nav-link" href="#">Recipe Finder</a>
+    <a id="title" class="col-8 flex-sm-fill text-sm-center nav-link align-middle" href="#">Bread and Ifits.</a>
+    <a class="col-2 flex-sm-fill text-sm-center nav-link" href="#" tabindex="-1" aria-disabled="true">Go to recipe
+        book</a>
+</nav>
+<div class="nav-transform"></div>
+<div class="container-fluid">
+    <div class="row" id="mainContent">
+        <aside class="col-2 justify-content-center">
+            <form>
+                <input class="choiceBtn col-2" id="getRecipeBtn" type="submit" value="Get Recipe">
+                <input class="choiceBtn" id="expandAll" type="submit" data-toggle="collapse"
+                       data-target=".multi-collapse" aria-expanded="false"
+                       aria-controls="multiCollapseExample1 multiCollapseExample2" value="Expand All">
```

Comment:
Expand all doesn't work? Sorry... :(

Reply:
Checked on both firefox and chrome

Reply:
This may be due to the fact that it's type="submit", as it appears to be refreshing the page each time we click on it.

Reply:
One test that we did was that we are able to remove type="submit", which appears to fix the issue. 

You can give it response to the user by giving it a cursor-type on the button, which will achieve the same effect as before without the bug.

Reply:
removed input type="submit"